stan/math/rev/prob.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef STAN_MATH_REV_PROB_HPP
+#define STAN_MATH_REV_PROB_HPP
+
+#include <stan/math/prim/fun/Eigen.hpp>
+
+#include <stan/math/rev/prob/std_normal_log_qf.hpp>
+
+#include <stan/math/prim/err.hpp>
+#include <stan/math/prim/fun.hpp>
+#include <stan/math/prim/functor.hpp>
+#include <stan/math/prim/prob.hpp>
+#include <stan/math/prim/meta.hpp>
```

Comment:
I think the `rev/prob` metaheader should only include the `prob` headers, as the individual headers should be including all other files that are needed